FHEM/14_SD_WS.pm

```diff
@@ -125,6 +127,49 @@ sub SD_WS_Undef {
   my ($hash, $name) = @_;
   carp "SD_WS_Undef, too few arguments ($hash, $name)" if @_ < 2;
   delete($modules{SD_WS}{defptr}{$hash->{CODE}}) if(defined($hash->{CODE}) && defined($modules{SD_WS}{defptr}{$hash->{CODE}}));
+	RemoveInternalTimer($hash);
+  return;
+}
+
+#############################
+sub SD_WS_Set {
+  my ($hash, $name, $cmd, @args) = @_;
+  my $ret = undef;
```

Comment:
Mich würde hier durchaus anlachen, die übergebenen Parameter zu valideren. Je nachdem, was halt erlaubt ist. 
In manchen subs hatten wir das auch schon begonnen (nicht in diesem Modul)

Reply:
Doch, auch in diesem Modul. Ich ergänze das.

Reply:
Hier hatte ich auch bereits ergänzt:

`  carp "SD_WS_Set, too few arguments ($hash, $name, $cmd)" if @_ < 3;
`

Reply:
Ahso, ich hatte mehr an so etwas gedacht:

```
my $hash = shift // carp 'device hash needs to be first parameter';
my $name= shift // carp 'name of the device must be second parameter';
my $cmd= shift // carp 'Specify the command as third prarameter';
my  @args = @_ // carp 'A minimum of one argument needs to be specified';
```

Reply:
OK, dann nehmen wir halt diese...